public/app.js

```diff
@@ -0,0 +1,330 @@
+// AUTH CONTROLLER
+var authController = (function() {
+    return {
+        //Authentication state listener
+        initApp: function() {
+            firebase.auth().onAuthStateChanged(function (user) {
+                if (user) {
+                    console.log('Login Successful');
```

Comment:
(appreciate this may have been inherited from previous code, but if could standardise on either "" or '' please)

Reply:
okay will go with double quotes